data/tlg0062/tlg001/tlg0062.tlg001.perseus-grc2.xml

```diff
@@ -1,22 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
  <teiHeader xml:lang="eng">
 <fileDesc>
 <titleStmt>
 <title xml:lang="grc">Φάλαρις</title>
 <author>Lucian of Samosata</author>
-<editor>A. M. Harmon</editor>
+<editor>Austin Morris Harmon, 1878-1950</editor>
```

Comment:
The years are not used, and we generally stick with the print author name, so the initials are what we would typically stick with. (As I did when I converted these files). Author name expansion is a catalog function.
When in doubt, the file header should look like the print.